todos/src/components/FilterButtons.js

```diff
@@ -1,13 +1,56 @@
-import React from 'react'
+import React from "react";
+import { setFilter } from "actions/index";
+import { useDispatch, useSelector } from "react-redux";
+
+const FilterButtons = () => {
+  const dispatch = useDispatch();
+  const filter = useSelector((state) => state.filterReducer.filter);
+  const ListLength = useSelector((state) => state.todoReducers.list).length;
+
+  const handleAllTodos = () => {
+    dispatch(setFilter("all"));
```

Comment:
Can we use const here?
```suggestion
    dispatch(setFilter(ALL));
```

Reply:
Yes, @hhu-wtag Bhai. Doing it right now.